service/Microsoft.Azure.Devices/Microsoft.Azure.Devices.csproj

```diff
@@ -289,7 +289,8 @@
     <Compile Include="Receiver.cs" />
     <Compile Include="RegistryManager.cs" />
     <Compile Include="RegistryStatistics.cs" />
-    <Compile Include="ServiceAuthenticationWithDeviceCredentials.cs" />
+    <Compile Include="ServiceAuthenticationWithDeviceSharedAccessPolicyToken.cs" />
```

Comment:
Note for -preview adopters: This is a public API breaking change between -preview packages. There is no expectation that API compat is maintained but existing applications may be impacted.

Workaround for apps using `ServiceAuthenticationWithDeviceCredentials` is to change to `ServiceAuthenticationWithDeviceSharedAccessPolicyKey`.